config/settings.json

```diff
@@ -171,4 +201,4 @@
       "type": "trueNorth"
     }
   }
-}
+}
```

Comment:
Were the non-whitespace changes here accidental?

Reply:
The ogcapi part was an inadvertent commit. I thought I reverted it all, but now see I missed some.

metrics is just whitespace:
```
    "metrics": {
      "enabled" : true
      "enabled": true
    },
```

Reply:
the ogcapi part should now be gone.